ui/src/app.jsx

```diff
@@ -55,6 +55,7 @@ export default React.createClass({
     '/teachermoments/csfair': 'messagePopupCsFair',
     '/teachermoments/bubblesort': 'messagePopupBubbleSort',
     '/teachermoments/jayden': 'jaydenScenario',
+    '/teachermoments/smith': 'SmithScenario',
```

Comment:
Can you make this camelCase like the others, where it starts with lowercase letters?